TYPO3.TYPO3CR/Tests/Functional/Domain/NodesTest.php

```diff
@@ -1199,12 +1277,17 @@ public function getPropertyDoesNotReturnNodeReferencesIfTheyAreNotVisibleAccordi
         $nodeC = $rootNode->createNode('node-c', $nodeType, 'e3b99700-f632-4a4c-2f93-0ad07eaf733f');
 
         $nodeA->setProperty('property2', '81c848ed-abb5-7608-a5db-7eea0331ccfa');
-        $nodeA->setProperty('property3', array('81c848ed-abb5-7608-a5db-7eea0331ccfa', 'e3b99700-f632-4a4c-2f93-0ad07eaf733f'));
+        $nodeA->setProperty('property3', [
+            '81c848ed-abb5-7608-a5db-7eea0331ccfa',
+            'e3b99700-f632-4a4c-2f93-0ad07eaf733f'
+        ]);
 
         $nodeB->setHidden(true);
 
         $this->assertNull($nodeA->getProperty('property2'));
-        $this->assertSame(array($nodeC), $nodeA->getProperty('property3'));
+        $property3 = $nodeA->getProperty('property3');
+        $this->assertCount(1, $property3);
+        $this->assertSame($nodeC, reset($property3));
```

Comment:
This had to be changed as the key numbering is no longer continuous due to the use of array_filter. This was never publicised as feature or expectation and won't be noticed in loops, just a direct comparison of the array with an expectation reveals it, therefore I adapted the assertions here.